tools/cli/admin.go

```diff
@@ -799,6 +803,55 @@ func newDBCommands() []cli.Command {
 				AdminDBScan(c)
 			},
 		},
+		{
+			Name:    "timers",
+			Aliases: []string{"timers"},
+			Usage:   "list timers",
+			Flags: append(getDBFlags(),
+				cli.IntFlag{
```

Comment:
nit: taking a list of shardIDs seems to be better/more versatile?

Reply:
you should easily generate a list of shards using some sub bash command and then fed into this command I assume?